### DISCUSSION TITLE
Table column tooltips

### DISCUSSION THREAD

**Discussion by OwKe** [filamentphp/filament#15106] [Ideas]
It would be nice to have the option to add tooltips to table column headers. Ideally allowing for html as well as a string.

Something like this: 

![image](https://github.com/user-attachments/assets/8b61094d-7daf-45fe-8b74-343b65ee1439)

**Comment by leandrocfe** [comment]
https://filamentglow.com/trick/add-tooltip-to-table-header-column-macro-0ecce75a

**Reply by OwKe** [reply]
Nice, thanks for this.
